### PR TITLE
feat: upgrade csi drivers

### PR DESCRIPTION
**Pull request by andyzhangx**
usually we want to keep 2 driver image versions in one vhd, after image bake happened, we could upgrade versions in aks rp later.

Latest azure disk version is v1.7.0, and latest azure file version is v1.6.0

### CODE REVIEW

**Review by ZeroMagic — APPROVED** [review]
/lgtm

File: vhdbuilder/packer/generate-windows-vhd-configuration.ps1 [hunk 1]
```diff
@@ -53,10 +56,13 @@ switch ($windowsSKU) {
             "mcr.microsoft.com/windows/nanoserver:1809",
             "mcr.microsoft.com/oss/kubernetes/pause:3.4.1",
             "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.3.0",
+            "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.4.0",
             "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.2.0",
             "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.3.0",
-            "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.4.0",
-            "mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:v1.4.0",
+            "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.5.0",
```
Comment (AbelHu) [inline vhdbuilder/packer/generate-windows-vhd-configuration.ps1:62]: Currently azuredisk-csi:v1.4.0 is not used but 1.5.0 is used, is it right?

Comment (mainred) [reply vhdbuilder/packer/generate-windows-vhd-configuration.ps1:62]: Agree, please make sure csi:v1.4.0 is not in production.

Comment (andyzhangx) [reply vhdbuilder/packer/generate-windows-vhd-configuration.ps1:62]: correct, it's just upgraded to v1.5.0 two weeks ago.

**Review by mainred — APPROVED** [review]

**Review by AbelHu — APPROVED** [review]

**Pull request merged**